README.md

```diff
@@ -1,5 +1,92 @@
 # Serverless Workflow Specification - Typescript SDK
 
-Provides (TODO: add specifics) for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
+Provides the Java API/SPI for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
```

Comment:
Could we maybe indicate that this is still WIP in the readme? 
Per your comment it seems there is still some work to be completed at a later point in time?

Reply:
Yes it is still work to do,

There is a _WIP_ two lines below. Do you want me to be more explicit? I am ok doing it, just want to be sure.

- [ ]  replace Java for Typescript

Reply:
Opps I missed that, sry. Yeah this is fine the way you have it.